<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>browserForm</class>
 <widget class="QWidget" name="browserForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="browser_address">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>15</y>
     <width>390</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="browser_back">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>11</y>
     <width>47</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="browser_forward">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>11</y>
     <width>47</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="browser_refresh">
   <property name="geometry">
    <rect>
     <x>637</x>
     <y>11</y>
     <width>47</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="browser_go">
   <property name="geometry">
    <rect>
     <x>578</x>
     <y>11</y>
     <width>47</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="browser_client" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>71</y>
     <width>705</width>
     <height>355</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
